tests/dragonfly/cluster_test.py

```diff
@@ -397,6 +396,11 @@ async def test_cluster_replica_sets_non_owned_keys(df_local_factory):
     except redis.exceptions.ResponseError as e:
         assert re.match(r"MOVED \d+ localhost:1111", e.args[0])
 
+    await push_config(replica_config, [c_master_admin])
+    await asyncio.sleep(0.5)
+    assert await c_master.execute_command("dbsize") == 0
+    assert await c_replica.execute_command("dbsize") == 0
+
```

Comment:
How about adding more complicated test that will check that the master and the replica delete only the relevant deleted slots. i.e use seeder to send data, peek a slot and update config with removing only this one slot, then check this key counter is 0 and validate the dbsize is equal to the dbsize before config update minus the number of keys in the removed slot

Reply:
That's a great idea. unfortunately I can't use seeder as it uses `MSET` (which returns `CROSSSLOTS` errors), but I'll just feed it with data in another way